src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/ApproveSpecificReleaseAuthorizationHandlersTests.cs

```diff
@@ -1,146 +0,0 @@
-using System;
```

Comment:
I see that we've removed the individual unit test classes for each individual Release Status handler as the various handlers are now consolidated into one, but do we now need to reinstate these tests against the new handler?

Reply:
Tests are in `ReleaseStatusAuthorizationHandlersTests.cs`, GitHub is probably just hiding it from you :wink:

Reply:
Bah it was!!!  Approved!